pkg/release/release.go

```diff
@@ -360,3 +365,102 @@ func CopyBinaries(rootPath string) error {
 	}
 	return nil
 }
+
+// WriteChecksums writes the SHA256SUMS/SHA512SUMS files (contains all
+// checksums) as well as a sepearete *.sha[256|512] file containing only the
+// SHA for the corresponding file name.
+func WriteChecksums(rootPath string) error {
+	logrus.Info("Writing artifact hashes to SHA256SUMS/SHA512SUMS files")
+
+	createSHASums := func(digest string) (string, error) {
+		fileName := fmt.Sprintf("SHA%sSUMS", digest)
+		file, err := os.Create(fileName)
+		if err != nil {
+			return "", errors.Wrapf(err, "create file %s", fileName)
+		}
+
+		if err := command.
+			New("find", rootPath, "-type", "f", "-printf", `%P\0`).
+			Pipe(
+				"xargs", "-0", "-I", "{}", "shasum", "-a",
+				digest, filepath.Join(rootPath, "{}"),
+			).
+			AddOutputWriter(file).
+			RunSuccess(); err != nil {
+			return "", errors.Wrap(err, "create SHA sum file contents")
+		}
+
+		return file.Name(), nil
+	}
```

Comment:
Any thoughts on using the `crypto` pkg for this rather than shelling out command?

Reply:
Done :+1: